specification/network/resource-manager/readme.md

```diff
@@ -250,6 +250,98 @@ csharp:
 ```
 
 
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: network
+  clear-output-folder: true
+```
+
+### Tag: package-2017-09 and go
+
+These settings apply only when `--tag=package-2017-09 --go` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-09' && $(go)
+output-folder: $(go-sdk-folder)/services/network/mgmt/2017-09-01/network
+```
+
+### Tag: package-2017-08 and go
+
+These settings apply only when `--tag=package-2017-08 --go` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-09' && $(go)
+output-folder: $(go-sdk-folder)/services/network/mgmt/2017-08-01/network
+```
+
+### Tag: package-2017-06 and go
+
+These settings apply only when `--tag=package-2017-06 --go` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-06' && $(go)
+output-folder: $(go-sdk-folder)/services/network/mgmt/2017-06-01/network
+```
+
+### Tag: package-2017-03 and go
+
+These settings apply only when `--tag=package-2017-03 --go` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-03' && $(go)
+output-folder: $(go-sdk-folder)/services/network/mgmt/2017-03-01/network
+```
+
+### Tag: package-2016-12 and go
+
+These settings apply only when `--tag=package-2016-12 --go` is specified on the command line.
+
+``` yaml $(tag) == 'package-2016-12' && $(go)
+output-folder: $(go-sdk-folder)/services/network/mgmt/2016-12-01/network
+```
+
+### Tag: package-2016-09 and go
+
+These settings apply only when `--tag=package-2016-09 --go` is specified on the command line.
+
+``` yaml $(tag) == 'package-2016-09' && $(go)
+output-folder: $(go-sdk-folder)/services/network/mgmt/2016-09-01/network
+```
+
+### Tag: package-2016-06 and go
+
+These settings apply only when `--tag=package-2016-06 --go` is specified on the command line.
+
+``` yaml $(tag) == 'package-2016-06' && $(go)
+output-folder: $(go-sdk-folder)/services/network/mgmt/2016-06-01/network
+```
+
+### Tag: package-2016-03 and go
+
+These settings apply only when `--tag=package-2016-03 --go` is specified on the command line.
+
+``` yaml $(tag) == 'package-2016-03' && $(go)
+output-folder: $(go-sdk-folder)/services/network/mgmt/2016-03-30/network
+```
+
+### Tag: package-2015-06split and go

```

Comment:
>06split [](start = 22, length = 7)

Why split? looks very weird

Reply:
this has to be there because it is the way that tag is declared higher up in the file.

Reply:
>n< Looks so weird. I want to know why the service team chose that tag